apps/labs/posts/design-of-the-versioned-hdf5-library.md

```diff
@@ -0,0 +1,182 @@
+---
+title: 'Design of the Versioned HDF5 Library'
+published: September 29, 2020
+author: aaron-meurer
+description: "In this post, we'll go into detail on how the underlying design of the library works on a technical level. Versioned HDF5 is a library that wraps h5py and offers a versioned abstraction for HDF5 groups and datasets."
+category: [PyData Ecosystem]
```

Comment:
```suggestion
category: [PyData ecosystem]
```

case sensitive, hoping [to make case insensitive](https://github.com/Quansight/Quansight-website/issues/634) in the future